### ISSUE TITLE
Confusing Information about sorting Results of Database Backend by Relevance

### ISSUE THREAD

**Issue by maqnius** [wagtail/wagtail#11353]
In the [docs](https://docs.wagtail.org/en/stable/topics/search/searching.html#search-operator) is stated that the database backend does not support ordering by relevance.

> We recommend using the “or” operator when ordering by relevance and the “and” operator when ordering by anything else **(note: the database backend doesn’t currently support ordering by relevance)**.

I guess and ticket #3312 indicates that this is not true anymore. So we should get rid of the note:

> We recommend using the “or” operator when ordering by relevance and the “and” operator when ordering by anything else.

Or is this still valid?

### Pertinent section of the Wagtail docs

https://docs.wagtail.org/en/stable/topics/search/searching.html#search-operator

**Comment by iamaamunir** [comment]
Hello. Can I be assigned to this issue?

**Comment by zerolab** [comment]
@iamaamunir thanks for the interest. please read https://docs.wagtail.org/en/stable/contributing/first_contribution_guide.html and https://docs.wagtail.org/en/stable/contributing/first_contribution_guide.html#contributing-a-solution in particular.

tl;dr - we do not assign issues to anyone outside of the core team. You have to read the whole issue thread and understand whether the issue requires work from someone in the core team (this one doesn't), whether there are any special considerations, and most importantly if there is a PR for the issue (in this case, there is #11364) and if there is, you can help with code review and testing, or you can find another issue
